backend/src/main/java/api/printing/PrintingController.java

```diff
@@ -0,0 +1,121 @@
+package api.printing;
+
+import api.ApiController;
+import model.DataModel;
+import model.DigitalPart;
+import model.DigitalPrint;
+import storage.repository.GenericRepository;
+import sun.reflect.generics.reflectiveObjects.NotImplementedException;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Use classes from the Java API instead of Sun classes. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1191)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'sun.reflect.generics.reflectiveObjects.NotImplementedException'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)